package/post-install.sh

```diff
@@ -139,12 +147,12 @@ println
 println "Logging $DD_TRACE_PHP_BIN -i to a file"
 println
 
-$DD_TRACE_PHP_BIN -i > "$EXTENSION_LOGS_DIR/php-info.log"
+invoke_php -i > "$EXTENSION_LOGS_DIR/php-info.log"
 
-PHP_VERSION=$($DD_TRACE_PHP_BIN -i | awk '/^PHP[ \t]+API[ \t]+=>/ { print $NF }')
-PHP_CFG_DIR=$($DD_TRACE_PHP_BIN -i | grep 'Scan this dir for additional .ini files =>' | sed -e 's/Scan this dir for additional .ini files =>//g' | head -n 1 | awk '{print $1}')
+PHP_VERSION=$(invoke_php -i | awk '/^PHP[ \t]+API[ \t]+=>/ { print $NF }')
+PHP_CFG_DIR=$(invoke_php -i | grep 'Scan this dir for additional .ini files =>' | sed -e 's/Scan this dir for additional .ini files =>//g' | head -n 1 | awk '{print $1}')
```

Comment:
Unrelated to this change, I noticed this chain of commands can be simplified:
 
```suggestion
PHP_CFG_DIR=$(invoke_php -i | awk -F"=>" '/Scan this dir for additional .ini files/ { sub(/^[ \t]/, "", $2) ; print $2 }')
```
A quick partial breakdown for those who don't know awk:
  - Regular expressions are delimited with `//`
  - The parameter `-F"=>"` sets the field separator to be `=>`, so the line gets broken by that instead of spaces.
  - `sub(/[^ \t]/, "", $2)` will do a substitution on the variable `$2`, in this case replace leading spaces and tabs with nothing (aka ltrim)

We use awk on the line above this, so we have `awk` available.

Reply:
Nice idea. Added as part of: APMPHP-210 once we improve verification.